packages/ui/components/topbanner/TopBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import { ExclamationIcon } from "@heroicons/react/outline";
+import { XIcon } from "@heroicons/react/solid";
+import classNames from "classnames";
+import { ReactNode, useState } from "react";
+
+export type TopBannerProps = {
+  text: string;
+  variant?: keyof typeof variantClassName;
+  actions?: ReactNode;
+};
+
+const variantClassName = {
+  default: "bg-gradient-primary",
+  warning: "bg-orange-400",
+  error: "bg-red-400",
+};
+
+export function TopBanner(props: TopBannerProps) {
+  const { variant = "default", text } = props;
+  const [isOpen, setOpen] = useState(true);
+
+  return (
+    <div
+      id="banner"
```

Comment:
If this is for testing the it should be a `data-testid` otherwise let's remove it since there may be multiple banners and create an ID conflicts

```suggestion
      data-testid="banner"
```